app/src/state/ducks/auth.js

```diff
@@ -282,3 +282,13 @@ export const rootState = state => state || {};
 export const isLoggedIn = state => {
   return state.auth.me !== null ? true : false;
 };
+
+export const isAdmin = state => {
+  return (
+    state.auth.me !==
+    null ? (
```

Comment:
Should this check entire array of permissions? or just first item? Does this need better error handling?